examples/landing-zone-v2/tier3/dns/dns.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# DNS A Record Example
+#########
+# A Record example for Client DNS subzone
+# SC-22
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: workload-name-client-name-standard-core-public-dns-a-rset # kpt-set: workload-name-${client-name}-standard-core-public-dns-a-rset
+  namespace: project-id-tier3 # kpt-set: ${project-id}-tier3
```

Comment:
namespace is not a required field because this resource is deployed in a repo-sync instead of a root-sync. by default all resources will be deployed in the default tier3 namespace

Reply:
Addressed. Please verify.